addons/mail/models/mail_render_mixin.py

```diff
@@ -395,14 +369,11 @@ def _render_template_jinja(self, template_txt, model, res_ids,
 
         :param dict add_context: additional context to give to renderer. It
           allows to add or update values to base rendering context generated
-          by ``MailRenderMixin._render_jinja_eval_context()``;
+          by ``MailRenderMixin._render_small_qweb_eval_context()``;
```

Comment:
Indeed, I just renamed jinja into qweb, I assumed that if somebody made a function there, it would have a purpose.
Do I remove it and use `_get_common_eval_context()` instead ?